lib/common/memory/src/mmap_ops.rs

```diff
@@ -137,12 +137,6 @@ pub fn transmute_to_u8<T>(v: &T) -> &[u8] {
     unsafe { std::slice::from_raw_parts(ptr::from_ref::<T>(v).cast::<u8>(), mem::size_of_val(v)) }
 }
 
-pub fn transmute_to_u8_mut<T>(v: &mut T) -> &mut [u8] {
```

Comment:
Great you found that this and `transmute_to_u8_slice_mut` are used only in gpu 🕵️‍♂️